x-pack/plugins/security_solution/public/explore/hosts/pages/navigation/host_risk_score_tab_body.tsx

```diff
@@ -93,6 +96,14 @@ export const HostRiskScoreQueryTabBody = ({
     isDeprecated: isDeprecated && !loading,
   };
 
+  if (!privileges.isLoading && !privileges.hasAllRequiredPrivileges) {
+    return (
+      <EuiPanel hasBorder>
+        <RiskEnginePrivilegesCallOut privileges={privileges} />
```

Comment:
I see this component receives the `privileges` and it does the same check again internally, why do we need that? 
If we only render this component when privileges are missing, the internal check is redundant, can we clean that?

Reply:
I mean, pass only `missingPrivileges`

Reply:
Good catch. 
The check probably made more sense previously, given how the component was being used. But now I think it does make sense to force the consumer to do the check 👍🏽

Reply:
Actually, scratch that. I found at least [one place ](https://github.com/tiansivive/kibana/blob/f52f550b5923d1d7fb2f76e2ac7d89b3099a0ae7/x-pack/plugins/security_solution/public/entity_analytics/pages/entity_analytics_management_page.tsx#L22) that relies on the internal check.
It might be that it makes sense to change that part as well, but I'd rather tackle that as a follow up